src/amo/components/ScreenShots/index.js

```diff
@@ -103,13 +55,36 @@ export default class ScreenShots extends React.Component<Props> {
             this.viewport = el;
           }}
         >
-          <PhotoSwipeGallery
-            className="ScreenShots-list"
-            close={this.onClose}
-            items={previews}
-            options={PHOTO_SWIPE_OPTIONS}
-            thumbnailContent={thumbnailContent}
-          />
+          <div className="ScreenShots-list">
+            <Gallery
+              options={PHOTO_SWIPE_OPTIONS}
+              onOpen={this.onOpenPhotoswipe}
+            >
+              {previews.map((preview) => (
+                <Item
+                  key={preview.src}
+                  original={preview.src}
+                  thumbnail={preview.thumbnail_src}
+                  width={preview.w}
+                  height={preview.h}
+                  title={preview.title}
+                >
+                  {({ ref, open }) => (
+                    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
+                    <img
+                      alt=""
```

Comment:
I notice that in the old code, the `img` has an `alt` attribute of `preview.title`. Do we not want that anymore? Is that dealt with via the `title` property of the `Item` above?

Reply:
I think setting `alt` to `preview.title` is a good idea. `title` was used for the caption in fullscrene but maybe `alt` could be used as well? Or we could have both. I'll update the PR, thanks!